src/GrpcProtos/capture.proto

```diff
@@ -300,7 +300,8 @@ message ClientCaptureEvent {
     SchedulingSlice scheduling_slice = 11;
     ThreadName thread_name = 12;
     ThreadStateSlice thread_state_slice = 13;
-    TracepointEvent tracepoint_event = 14;
+    TotalMemoryUsageInfo total_memory_usage_info = 14;
+    TracepointEvent tracepoint_event = 15;
```

Comment:
when adding new field to proto message do not change field number for existing ones. They are needed for backwards compatibility.

Reply:
Thanks for telling me about this! Changed accordingly.